robottelo/ui/locators.py

```diff
@@ -1,14 +1,35 @@
 # -*- encoding: utf-8 -*-
-# vim: ts=4 sw=4 expandtab ai
+"""Implements different locators for UI"""
 
-"""
-Implements different locators for UI
-"""
+import collections
+import logging
 
 from selenium.webdriver.common.by import By
 
 
-menu_locators = {
+LOGGER = logging.getLogger(__name__)
+
+
+class LocatorDict(collections.Mapping):
+    """This class will log every time an item is selected"""
+    def __init__(self, *args, **kwargs):
+        self.store = dict(*args, **kwargs)
```

Comment:
Would it be simpler to just accept a single argument? For example:

```python
def __init__(self, dictionary):
    self.store = dictionary
```

Or are there techniques for dictionary creation that necessitate that `*args` and `**kwargs` be accepted instead?